compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -585,6 +585,8 @@ void addMmt4dTilingExpertPassPipeline(OpPassManager &passManager,
 void addCPUDataTilingPipeline(OpPassManager &passManager) {
   addTileAndDistributePasses(passManager);
   OpPassManager &nestedModulePM = passManager.nest<ModuleOp>();
+  nestedModulePM.addNestedPass<func::FuncOp>(
```

Comment:
This will affect all MMT4D ops?

Reply:
No, it won't affect mmt4d ops. The mmt4d ops go with `Mmt4dTilingExpertPassPipeline`